apps/wallet-dashboard/components/Dialogs/SendToken/views/SentSuccessView.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { useGetTransaction } from '@iota/core';
+import { InfoBoxType, InfoBox, InfoBoxStyle } from '@iota/apps-ui-kit';
+import { Warning } from '@iota/ui-icons';
+import { getExtendedTransaction } from '@/lib/utils';
+import { useCurrentAccount } from '@iota/dapp-kit';
+import { TransactionDialogDetails } from '../../transaction';
+
+interface SentSuccessProps {
+    digest?: string;
+    onClose: () => void;
+}
+
+export function SentSuccessView({ digest, onClose }: SentSuccessProps) {
+    const currentAccount = useCurrentAccount();
+    const { data, isError, error } = useGetTransaction(digest || '');
+
+    const transaction = data && getExtendedTransaction(data, currentAccount?.address || '');
+
+    if (isError) {
+        return (
+            <InfoBox
+                type={InfoBoxType.Error}
+                title="Something went wrong"
+                supportingText={error?.message ?? 'An error occurred'}
```

Comment:
custom the error message

Reply:
something like
`title="Error getting transaction info"`
`supportingText={error?.message ?? 'An error occurred when getting the transaction info'}`